jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java

```diff
@@ -30,6 +30,7 @@
 import static com.epam.jdi.light.common.SearchStrategies.*;
 import static com.epam.jdi.light.driver.WebDriverByUtils.*;
 import static com.epam.jdi.light.elements.base.JdiSettings.*;
```

Comment:
please revert all changes in this file
None of them required for mobile

Reply:
revert master

Reply:
reverted